src/bundle/Resources/public/scss/_edit-header.scss

```diff
@@ -8,7 +8,7 @@
     border-top-left-radius: $ibexa-border-radius;
     border-top-right-radius: $ibexa-border-radius;
     transition: all $ibexa-admin-transition-duration $ibexa-admin-transition, border-bottom-width 0;
-    z-index: 2;
+    z-index: 1050;
```

Comment:
Note: restored after mistakenly deleted in: https://github.com/ibexa/admin-ui/pull/886/files#diff-4b3ad942bc4ac8d54e34e612e6b4708697317a9ae52700f12e676e2ebc1baa7b